google/oauth2/_client.py

```diff
@@ -108,7 +108,13 @@ def _token_endpoint_request(request, token_uri, body):
             if hasattr(response.data, "decode")
             else response.data
         )
-        response_data = json.loads(response_body)
+        try:
+            response_data = json.loads(response_body)
+        except (KeyError, ValueError):
```

Comment:
```suggestion
        except json.DecodeError
```

Reply:
That would be a Python3-only change, i think.